SimCalorimetry/HGCalSimProducers/python/hgcalDigitizer_cfi.py

```diff
@@ -273,3 +273,8 @@ def HGCal_disableNoise(process):
 )
 phase2_hgcalV9.toModify(HGCAL_noise_fC, values = [x*fC_per_ele for x in nonAgedNoises_v9])
 phase2_hgcalV9.toModify(HGCAL_noises, values = [x for x in nonAgedNoises_v9])
+
+from Configuration.Eras.Modifier_phase2_hfnose_cff import phase2_hfnose
```

Comment:
The `phase2_hfnose` modifier is not just used with the v9 HGCal, so this is not correct. It should also be unnecessary, since when it is used with the v9 HGCal, the `phase2_hgcalV9` modifications will already be picked up.

If it is necessary for some reason I'm missing, then rather than copying the modifications, just change the lines above this one to: `(phase2_hgcalV9 | phase2_hfnose).toModify(...)`

Reply:
ok I agree the pset is already replaced for both HGC and HFnose, so it's unnecessary at this stage. I removed the commit